spec/spec_helper.rb

```diff
@@ -42,20 +42,20 @@ def set_node_version(version)
   end
 end
 
+def resolve_binary_path
+  (/darwin/ =~ RUBY_PLATFORM) != nil ? './vendor/resolve-version-darwin' : './vendor/resolve-version-linux'
```

Comment:
The logic here is a bit tricky, I think we want to say that if we make a match we want to use darwin, however the `!= nil` takes a second or two to parse out. Might do something like:

```
   RUBY_PLATFORM.match(/darwin/) ? './vendor/resolve-version-darwin' : './vendor/resolve-version-linux' 
```